README.md

```diff
@@ -77,7 +77,9 @@ pacman -S gopass
 [![Scoop version](https://img.shields.io/scoop/v/gopass)](https://github.com/ScoopInstaller/Main/blob/master/bucket/gopass.json)
 
 ```shell
-# WinGet
+# WinGet#
```

Comment:
The trailing `#` shouldn't be there, I think.

Reply:
oops, you're right!

```suggestion
# WinGet
```